source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -124,6 +124,7 @@
   "global.duration.months": "月",
   "global.duration.seconds": "秒",
   "global.duration.years": "年",
+  "global.errors.failureAlert": "!!!Failure Alert",
```

Comment:
Can you add JP translations?

Reply:
@DeeJayElly This is out of scope of this PR.